vcpkg-configuration.json

```diff
@@ -0,0 +1,10 @@
+{
+  "overlay-ports": [
+    "overlay/ports",
+    "overlay/osx",
+    "overlay/windows"
+  ],
+  "overlay-triplets": [
+    "overlay/triplets"
+  ]
+}
```

Comment:
Another note: Using a separate `vcpkg-configuration.json` should make sure that all vcpkg commands respect this configuration, including e.g. `depend-info`, regardless of whether they are manifest-related or not.